### DISCUSSION TITLE
Placing the add command at the top bar (or outside the list panel)

### DISCUSSION THREAD

**Discussion by johardi** [redivis/meta#175] [Feature requests]
**What is the problem you'd like us to address**
It's frustrating when adding a new table or importing a new file because the command button is at the bottom of the page, especially if the table or file list is long. You need to scroll down a lot before getting to the command button.

**Describe a solution (if you have one in mind)**
Perhaps if the command button is at the top, or outside the list panel, then the need to scroll down is very minimal.

**Describe other related products we could check out (if applicable)**
N/A

**Any additional context**
N/A

**Comment by erindelaney** [comment]
Thanks @johardi! We noted this internally and will work on making this interaction easier.

**Comment by imathews** [comment]
This was fixed with comprehensive updates to lists throughout Redivis
